### PR TITLE
Create example app with pipeline-ui and react

### PR DESCRIPTION
**Pull request by rmeena840**
increase-counter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "monitor-payments",
     "shop",
     "warcraft/dapp",
-    "wallet"
+    "wallet",
+    "pipeline-ui"
```
Comment (vuvoth) [inline package.json:15]: I think you forgot update yarn.lock in root project!

File: pipeline-ui/src/react-app-env.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (vuvoth) [inline pipeline-ui/src/react-app-env.d.ts]: What is this file?

File: pipeline-ui/src/teal/approval_program.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#pragma version 6
```
Comment (vuvoth) [inline pipeline-ui/src/teal/approval_program.txt:1]: I think we should rename it to .teal

**Review by vuvoth — COMMENTED** [review]
Quick review!

File: pipeline-ui/src/teal/approval_program.txt (hunk 3 above)
Comment (rmeena840) [reply pipeline-ui/src/teal/approval_program.txt:1]: pipeline needs it txt file as props. For more info: https://www.pipeline-ui.com/docs/AlgoComponents/AlgoDeploy

File: pipeline-ui/src/react-app-env.d.ts (hunk 2 above)
Comment (rmeena840) [reply pipeline-ui/src/react-app-env.d.ts]: This file was automatically added while initializing react project.

File: package.json (hunk 1 above)
Comment (rmeena840) [reply package.json:15]: This is a workspace, not a dependency.

File: pipeline-ui/src/react-app-env.d.ts (hunk 2 above)
Comment (rmeena840) [reply pipeline-ui/src/react-app-env.d.ts]: Removed it.

File: pipeline-ui/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
```
Comment (Megha-Dev-19) [inline pipeline-ui/public/manifest.json]: I don't think we need the react logo, and we can update the name an everything else as per our app config

File: pipeline-ui/src/App.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.App {
+	text-align: center;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
+}
```
Comment (Megha-Dev-19) [inline pipeline-ui/src/App.css:7]: does pipeline-ui has there own components.? may be we can use them

File: pipeline-ui/src/components/IncreaseCounter.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+interface IncreaseCounterProps {
+	addr: string;
+	appId: number;
+}
+
+class IncreaseCounter extends Component<IncreaseCounterProps, {}> {
+	appCall = () => {
+		const networkType = Pipeline.main
+			? NetworkType.MAIN_NET
+			: NetworkType.TEST_NET;
```
Comment (Megha-Dev-19) [inline pipeline-ui/src/components/IncreaseCounter.tsx:16]: In the UI screenshot, I can only see the main net config

File: pipeline-ui/src/components/ReadAppData.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+			) {
+				this.setState({ counter: res[0]?.value?.uint });
+			}
+		}, 10000);
+	}
+
+	render() {
+		return (
+			<>
+				<h4>{'Counter: ' + this.state.counter}</h4>
```
Comment (Megha-Dev-19) [inline pipeline-ui/src/components/ReadAppData.tsx:37]: Can you help me understand what is the use of counter?

File: pipeline-ui/src/components/WalletConnect.tsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+const { AlgoAddress, Pipeline, AlgoSignerButton } = require('pipeline-ui');
+
+interface WalletConnectProps {
+	updateAddress: Function;
+}
+
+interface WalletConnectState {
+	Algaddress: string;
```
Comment (Megha-Dev-19) [inline pipeline-ui/src/components/WalletConnect.tsx]: what's `Algaddress`?

**Review by Megha-Dev-19 — COMMENTED** [review]
Reviewed.

File: pipeline-ui/src/App.css (hunk 5 above)
Comment (rmeena840) [reply pipeline-ui/src/App.css:7]: They don't have anything like container like we have in antd. This is required.

File: pipeline-ui/src/components/IncreaseCounter.tsx (hunk 6 above)
Comment (rmeena840) [reply pipeline-ui/src/components/IncreaseCounter.tsx:16]: Unchecking the checkbox will be treated as TestNet. This component is from pipeline.

File: pipeline-ui/src/components/ReadAppData.tsx (hunk 7 above)
Comment (rmeena840) [reply pipeline-ui/src/components/ReadAppData.tsx:37]: The template name is 'Increment Counter App'. On calling proposal we are increasing the counter.

File: pipeline-ui/src/components/WalletConnect.tsx (hunk 8 above)
Comment (rmeena840) [reply pipeline-ui/src/components/WalletConnect.tsx]: Pipeline documentation had suggested to use this name. I have changed it to just `address`

File: pipeline-ui/public/manifest.json (hunk 4 above)
Comment (rmeena840) [reply pipeline-ui/public/manifest.json]: Removed it.

File: pipeline-ui/src/App.css (hunk 5 above)
Comment (Megha-Dev-19) [reply pipeline-ui/src/App.css:7]: okay

File: pipeline-ui/src/components/IncreaseCounter.tsx (hunk 6 above)
Comment (Megha-Dev-19) [reply pipeline-ui/src/components/IncreaseCounter.tsx:16]: This is a little weird for me, but thanks for informing

File: pipeline-ui/src/components/WalletConnect.tsx (hunk 8 above)
Comment (Megha-Dev-19) [reply pipeline-ui/src/components/WalletConnect.tsx]: thanks

**Review by vuvoth — APPROVED** [review]

**Pull request merged**